### PR TITLE
Updated "Scrollable content can be reached with sequential focus navigation" [0ssw9k]: scrollable regions pass when there is an open modal

### PR DESCRIPTION
**Pull request by giacomo-petri**
Closes issue(s): #2217

This update addresses scrollable elements that become inert, such as when overlaid by modals. The update includes:
- A revision of the expectation to exclude inert scrollable elements from sequential focus.
- An added passing example that illustrates this adjustment.

Need for Call for Review:
This will require a 2 weeks Call for Review

---

### **When creating PR:**

- [ ] Make sure you're requesting to **pull a branch** (right side) to the `develop` branch (left side).
- [ ] Make sure you do not remove the "How to Review and Approve" section in your pull request description

### **After creating PR:**

- [ ] Add yourself (and co-authors) as "Assignees" for PR.
- [ ] Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- [ ] [Link the PR to any issue it solves](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue). This will be done automatically by [referencing the issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) at the top of this comment in the indicated place.
- [ ] Optionally request feedback from anyone in particular by assigning them as "Reviewers".

### **When merging a PR:**

- [ ] Close any issue that the PR resolves. This will happen automatically upon merging if the PR was correctly linked to the issue, e.g. by referencing the issue at the top of this comment.

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Call for Review period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: _rules/scrollable-element-keyboard-accessible-0ssw9k.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ This rule applies to any [HTML element][] that has [visible][] [children][] in t
 
 ## Expectation
 
-Each test target is either included in [sequential focus navigation][] or has a [descendant][] in the [flat tree][] that is included in [sequential focus navigation][].
+Each test target is either included in [sequential focus navigation][] or has a [descendant][] in the [flat tree][] that is included in [sequential focus navigation][], unless the test target is [inert][].
```
Comment (Jym77) [inline _rules/scrollable-element-keyboard-accessible-0ssw9k.md]: **Suggestion:** should we switch that to an alternative list?
```suggestion
For each target element, at least one of the following is true: 
- the element is included in [sequential focus navigation][]; or 
- the element has a [descendant][] in the [flat tree][] that is included in [sequential focus navigation][]; or
- the element is [inert][].
```

Comment (Jym77) [inline _rules/scrollable-element-keyboard-accessible-0ssw9k.md]: **Issue:** Looks like the link to `[inert]` is missing.

**Review by Jym77 — CHANGES_REQUESTED** [review]
Looks good. Missing a link.

File: _rules/scrollable-element-keyboard-accessible-0ssw9k.md [hunk 2]
```diff
@@ -108,6 +111,46 @@ This [scrollable][] `section` element contains a link that is included in [seque
...
+	</p>
+</section>
+<dialog id="ppDialog" aria-labelledby="dialogLabel">
+	<h2 id="dialogLabel">WCAG 2.2</h2>
+	<p>
+		<a href="https://www.w3.org/TR/WCAG22/">WCAG 2.2</a>
+	</p>
+	<button id="cancel" onclick="ppDialog.close()">Cancel</button>
+</dialog>
+<script>
```
Comment (carlosapaduarte) [inline _rules/scrollable-element-keyboard-accessible-0ssw9k.md:144]: Do we need this line (and line 146)?
I don't think so, but just wanting to make sure.

**Review by carlosapaduarte — APPROVED** [review]
Left a comment about two lines in the example that don't seem to be needed.

File: _rules/scrollable-element-keyboard-accessible-0ssw9k.md [hunk 3]
```diff
@@ -108,6 +111,46 @@ This [scrollable][] `section` element contains a link that is included in [seque
...
+	const openDialog = () => {
+		ppDialog.showModal();
+		myFrame.tabIndex = '-1'
```
Comment (carlosapaduarte) [inline _rules/scrollable-element-keyboard-accessible-0ssw9k.md:147]: Do we need this line (and line 149)?
I don't think so, but just wanting to make sure.

File: _rules/scrollable-element-keyboard-accessible-0ssw9k.md (hunk 2 above)
Comment (giacomo-petri) [reply _rules/scrollable-element-keyboard-accessible-0ssw9k.md:144]: The intention was to create a real case scenario, but I’m fine with removing them if we believe they could cloud the outcome.

File: _rules/scrollable-element-keyboard-accessible-0ssw9k.md (hunk 3 above)
Comment (giacomo-petri) [reply _rules/scrollable-element-keyboard-accessible-0ssw9k.md:147]: The intention was to create a real case scenario, but I’m fine with removing them if we believe they could cloud the outcome.

**Review by Jym77 — APPROVED** [review]

**Review by WilcoFiers — APPROVED** [review]

**Comment by giacomo-petri** [comment]
Call for review ends November 20th

**Pull request merged**